cmd/osbuild-composer/main.go

```diff
@@ -118,11 +119,28 @@ func main() {
 
 	store := store.New(&stateDir)
 
-	workerAPI := worker.NewServer(logger, store)
-	weldrAPI := weldr.New(rpm, arch, distribution, repoMap[common.CurrentArch()], logger, store)
+	queueDir := path.Join(stateDir, "queue")
+	err = os.Mkdir(queueDir, 0700)
+	if err != nil && !os.IsExist(err) {
+		log.Fatalf("cannot create queue directory: %v", err)
+	}
+
+	jobs, err := fsjobqueue.New(queueDir)
+	if err != nil {
+		log.Fatalf("cannot create jobqueue: %v", err)
+	}
+
+	outputDir := path.Join(stateDir, "outputs")
+	err = os.Mkdir(outputDir, 0755)
+	if err != nil && !os.IsExist(err) {
+		log.Fatalf("cannot create output directory: %v", err)
+	}
+
+	workers := worker.NewServer(logger, jobs, store.AddImageToImageUpload)
```

Comment:
As discussed, we agreed that we can get away with not having the callback, but let weldr query the queue, right?

Reply:
Yeah, but I hope that this can be done in a follow-up?

Reply:
Agreed